_includes/identity/icons/index.md

```diff
@@ -20,7 +20,7 @@
 | {% include icons/error.svg %} | {% include icons/error-round.svg %} | error | delete, close, remove, multiply, multiplication, x |
 | {% include icons/warning.svg %} | {% include icons/warning-round.svg %} | warning | alert, exclamation-mark |
 | {% include icons/help.svg %} | {% include icons/help-round.svg %} | help | question, question-mark |
-| {% include icons/update.svg %} | {% include icons/update-round.svg %} | update |  |
+| {% include icons/update.svg %} | {% include icons/update-round.svg %} | update | updating(animated) |
```

Comment:
It's confusing to me to include "animated" here when it's not technically a part of the alias. 

Options to improve:
- If we're calling attention to a specific alias for triggering an animated state, let's be explicit about that. Something like 'updating (_used for animated state_)' or something.
- If 'animated' is just actually one of the aliases, let's format it the same as the other icons by separating the list of aliases with commas.
- If we decide to keep the phrasing and formatting almost exactly as is, it's missing a space before the first parenthesis. 😎

Reply:
Thank you, fixed and pushed.